task/buildah.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: buildah
```

Comment:
Is there any difference between this task and the tasks in https://github.com/tektoncd/catalog ?

Reply:
They're similar, yes, but there are still differences, like the default Dockerfile path, TLSVerify value, and working dir.